pepper/libpepper.py

```diff
@@ -361,6 +358,27 @@ def runner(self, fun, **kwargs):
 
         return self.low([low], path='/')
 
+    def wheel(self, fun, arg=None, **kwargs):
+        '''
+        Run a single command using the ``wheel`` client
+
+        Usage::
+          wheel('key.accept', match='myminion')
+        '''
+        low = {
+            'client': 'wheel',
+            'fun': fun,
+        }
+
+        if arg:
+            low['arg'] = arg
+        # if kwarg:
```

Comment:
WheelClient's cmd_sync/cmd_async methods definitely accept kwargs. I kind of remember they do **not** accept positional args though...